modules/tinymce/src/plugins/fullscreen/main/ts/Plugin.ts

```diff
@@ -10,10 +10,11 @@ import PluginManager from 'tinymce/core/api/PluginManager';
 import * as Api from './api/Api';
 import * as Commands from './api/Commands';
 import * as Buttons from './ui/Buttons';
+import Editor from 'tinymce/core/api/Editor';
 
 export default function () {
-  PluginManager.add('fullscreen', (editor) => {
-    const fullscreenState = Cell(null);
+  PluginManager.add('fullscreen', (editor: Editor) => {
```

Comment:
The editor type isn't needed here, as it's inferred since `PluginManager` is typed.